src/layout/RootLayout.tsx

```diff
@@ -10,7 +10,6 @@ import '@/css/globals.css';
 import { Auth0Provider } from '@/lib/auth0-provider';
 import { isProd, isStaging } from '@/helpers/env';
 
-Settings.throwOnInvalid = true;
```

Comment:
unrelated but just discovered this causes exceptions when in transaction form you type invalid year (i.e. 5 digits).